Steer/DigitizerWorkflow/src/FDDDigitizerSpec.cxx

```diff
@@ -70,6 +83,12 @@ class FDDDPLDigitizerTask : public o2::base::BaseDPLDigitizer
       LOG(info) << "FDD TIME RECEIVED " << record.getTimeNS();
     }
 
+    // Initialize the dead channel map
+    if (mUpdateDeadChannelMap && mUseDeadChannelMap) {
+      auto deadChannelMap = pc.inputs().get<o2::fit::DeadChannelMap*>("fdddeadchannelmap");
+      mDigitizer.setDeadChannelMap(deadChannelMap.get());
```

Comment:
This will work, but the good way of setting the ccdb object is to just call here 
```
if (mUpdateDeadChannelMap && mUseDeadChannelMap) {
  mDigitizer.setDeadChannelMap(deadChannelMap.get());
}
```
and at https://github.com/AliceO2Group/AliceO2/pull/12340/files#diff-805a3dc073d4a3b3a39103b840d1ca9560089a7a03ff59878d6ad06a51b2cab3R62 to do
```
    if (matcher == ConcreteDataMatcher("FDD", "DeadChannelMap", 0)) {
      mUpdateDeadChannelMap = false;
      mDigitizer.setDeadChannelMap((const o2::fit::DeadChannelMap*)obj);
    }
```